Rakefile

```diff
@@ -33,6 +33,15 @@ task release_all: [:webpack] do
   end
 end
 
+desc "Makes sure all official locales are complete and clean."
+task :check_locale_completeness do
+  DECIDIM_GEMS.each do |gem_name|
+    Dir.chdir("#{File.dirname(__FILE__)}/decidim-#{gem_name}") do
+      system({ "ENFORCED_LOCALES" => "en,ca,es" }, "rspec spec/i18n_spec.rb")
```

Comment:
No problem! Yes, it's arbitrary. So there's two ideas here:

* One is trying to keep locales in core that are currently 100% complete, 100% complete. This would account for the concept of "officially supported locales". Since I guess most core developers right now speak all three languages and most current installations include `ca` and `es`, choosing `es`, `ca` and `en` as official locales seems natural. But yes, this concept of official locales is a bit arbitrary and would probably no longer make sense once/if we extract `decidim-i18n` to a separate library.

* The other one is to allow installations to check their own locales. I think there's no doubts about this, we all want it, right?